101-workshop-nodejs/ex0-what-is-serverless/serverless-defined.md

```diff
@@ -17,19 +17,11 @@
 #
 -->
 
-# What is Serverless?
+# What is serverless?
 
-## The marketing hype
+It's most likely that the term _serverless_ was coined to attract the attention of developers wishing to reduce the server costs of running their applications. Although the name implies that code can be run without the need for classic servers, that simply is not true!
 
-Likely, thanks to some _marketing person_, the term _"Serverless"_ was coined to attract the attention of people wishing reduce the server costs of running their applications.
-
-{% hint style="warning" %}
-The name implies that code can be run without the need for classic servers _**which is absolutely not true**_!
-{% endhint %}
-
-## The ideal of Serverless
```

Comment:
Would want to leave the title if possible

Reply:
I don't think that title is grammatically correct or sounds right. And we removed the other headings/titles. 

The overall title is "Definition" and it, to me, all the content flowed fine under that overall title so there didn't seem to be a need for subheadings on this page.